tests/jballerina-unit-test/src/test/resources/test-src/statements/variabledef/configurable_global_var_decl_negative_03.bal

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// configurable var is implicitly  readonly
+int[] nonReadOnlyArray = [];
+
+configurable int[][] x = [nonReadOnlyArray];
```

Comment:
Shall we add another negative test for when the value itself is not immutable (as opposed to a member).

e.g.,
```ballerina
type Foo record {|
    int i;
|};

Foo foo = {i: 1};

configurable Foo x = foo;
```